engine/class_modules/priest/sc_priest.hpp

```diff
@@ -776,7 +775,7 @@ struct priest_action_t : public Base, public parse_buff_effects_t<priest_td_t>
   {
     // using S = const spell_data_t*;
 
-    parse_buff_effects( p().buffs.voidform );
+    parse_buff_effects( p().buffs.voidform, 0x4U, false, false );  // Skip 3 for Ancient Madness
```

Comment:
We use the masks to apply different rules for the different effects. We effectively parse buff effects twice for DA and VF:
1. Parse the original spell for the damage multiplier. Dont use stacks for this and mask off the effect for AM
2. Parse the original spell _again_ for the ancient madness passlist. Mask off the damage multiplier so it isnt applied twice and use stacks and the default value for this to get the crit increase from the spell.